src/providers/LocalVideoProvider/index.tsx

```diff
@@ -68,7 +68,7 @@ const LocalVideoProvider: React.FC = ({ children }) => {
       audioVideo?.stopLocalVideoTile();
```

Comment:
Checking if we need to call `stopVideoInput` and then `stopLocalVideoTile` or just `stopLocalVideoTile` is fine?

Reply:
Confirmed with Trung that we should call `stopVideoInput` instead of `stopLocalVideoTile`. In JS SDK `stopVideoInput` kills stream and also stops local video tile. No need to call both of them.

So we find a bug in JS SDK. Currently we just call `stopLocalVideoTile()` here and it's supposed not to kill the media stream, but it does now.That's why I didn't find this code is wrong during my testing.

Updated.